.github/workflows/charm-release.yaml

```diff
@@ -70,7 +70,9 @@ jobs:
             sudo iptables -P FORWARD ACCEPT
           fi
       - name: Upload charm to CharmHub
-        uses: canonical/charming-actions/upload-charm@2.4.0
+        # Using the version of charming-actions that supports release of multiple builds at once.
+        # The commit hash should be replaced by release version after the next release of charming-actions.
+        uses: canonical/charming-actions/upload-charm@0953ad46cdd8363b3c9c923793c467c5d948ce7f
```

Comment:
I would not be easier to use main directly?

Reply:
I thought it might be better to pin down a version like done here rather than tracking `main` directly.

Reply:
It is to avoid a future commit breaking any functionality required for our release to work. This commit's behaviour has been verified manually, but we can't say that about all the future commits if we directly track main.

Reply:
Yes, it's like using edge channel. I think it's ok to use main and if we found a bug on main, we will report it and use hot fix branch or some commit.

Reply:
I have used a recent pre-release version `2.5.0-rc` which was just made today.